ttbd/ui/templates/widget-target-tunnels.html

```diff
@@ -65,6 +65,96 @@ <h4 class='info info-grey'>
       </tr>
       {% endfor %}
 
+      <!-- Add row -->
+      <tr>
+        <td>
+          <div>
+            <select name="proto" required id = "input_field_id_tunnel_add_protocol">
```

Comment:
oooh so the variable in the javascript `input_field_id` literally is `"input_field_id"` then maybe we can remove it as a variable and hardcode it?

Reply:
I made it explicit so it is clear what it refers to -- I am finding I get lost, since many things have the same  "name" but they are used for different things surrounding that name, so namespacing that is helping me keep track.